Bastien Olivier Dijkstra/Clean.cs

```diff
@@ -0,0 +1,58 @@
+﻿using UnityEngine;
+
+public class Clean : MonoBehaviour {
+
+    public float range = 100f;
```

Comment:
While your code looks good and the brackets are all in the right places now, I noticed you haven't made use of Properties, which is something you could take a look at.

Reply:
FINE! :P